### ISSUE TITLE
I want to remove the option when uninstalling the plugin, but I encountered a problem

### ISSUE THREAD

**Issue by jeffwudev** [ko31/simple-slug-translate#33]
i try to add some code to delete option when plugin uninstall.
`register_uninstall_hook( __FILE__,  array( $this, 'register_uninstall_hook') );`
```php
public function register_uninstall_hook() {
    delete_option( $this->option_name );
}
```
but i got the error:
`register_uninstall_hook was called incorrectly. Only a static class method or function can be used in an uninstall hook.`
i don't know what the reason is, I hope to get your help

PHP Version: 7.3.12

**Comment by ko31** [comment]
@jeff-wu 
It might be easier to use `uninstall.php`.

```
<?php
// if uninstall.php is not called by WordPress, die
if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
	die;
}

$option_name = 'simple-slug-translate';

delete_option( $option_name );

delete_site_option( $option_name );
```
https://developer.wordpress.org/plugins/plugin-basics/uninstall-methods/#method-2-uninstall-php

If you can improve it, please make a PR!

**Comment by ko31** [comment]
@wujiechn 

I released version 2.7.0, and it contains `uninstall.php`.
https://wordpress.org/plugins/simple-slug-translate/

Thank you for your opinion :pray:
